processing/logs.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import multiprocessing
+from logging.handlers import QueueListener, QueueHandler
+
+
+def init_master():
+    q = multiprocessing.Queue()
+
+    handler = logging.StreamHandler()
+    # handler.terminator = "\n\r"
```

Comment:
remove this

Reply:
Oops. That was left over from when I was troubleshooting something.